### ISSUE TITLE
Move repo to official place (gitlab at https://gitlab.nic.cz/turris)

### ISSUE THREAD

**Issue by emanuelb** [jose1711/qemu_turrisos#4]
I found this repo from documentation at:
https://doc.turris.cz/doc/en/public/running_in_qemu#running_turris_os_3_inside_qemu
which state at the beginning:
>  If you want to run unmodified TOS or TOS4/TOS5, check out https://github.com/jose1711/qemu_turrisos 

I expect to find official code related to TOS in Turris project/repo at:
https://gitlab.nic.cz/turris
This way it's also auto-increase the trust assumed for this repo (as it's in official location)

**Comment by jose1711** [comment]
Fair enough but I have never received a blessing from TOS developers (the only response I got from them was that they never really tried in QEMU but it may work as OpenWrt does work this way). Not really sure if they're interested.

**Comment by emanuelb** [comment]
Thanks for your work/script, I assumed you from TOS team, still think it's good idea to ask them again? (in forum / gitlab) as it's very useful to be able to run TOS in QEMU.

**Comment by jose1711** [comment]
Well.. feel free. To be honest my only motivation was to test before I make a switch from TOS3 to TOS5 (still on 3 btw) and maybe check what features I may be missing. Testing on virtual hardware has its limits (memory size included) so not sure if this can be taken seriously.
